flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
Is aarch64 available? Can you build from source?

Reply:
this question hasn't been answered.

Reply:
will add it once stabilize x64 first

Reply:
what is it to stabilize? What I just read is "it won't happen"